_rules/SC3-1-1-html-lang-valid.md

```diff
@@ -19,13 +19,13 @@ authors:
 
 ### Applicability
 
-The root element of the page, if it is an `html` element with a `lang` and/or `xml:lang` attribute.
+The root element of the page, if it is an `html` element with a [non-empty](#non-empty) `lang` and/or `xml:lang` attribute.
 
 ### Expectation
 
 The value of the `lang` or `xml:lang` attribute conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
 
-**Note**: If `lang` and `xml:lang` exists, then only `lang` is tested for conformance.
+**Note**: If both `lang` and `xml:lang` exists, then only `lang` is tested for conformance.
```

Comment:
In the event of parser presence - what takes precedence - xml:lang is expected - but this is polymorph HTML5 if xml:lang is present at all which implies that a parser IS present. @jeankaplansky to investigate from the XML parser context. This applies to https://github.com/auto-wcag/auto-wcag/pull/107, https://github.com/auto-wcag/auto-wcag/pull/109, and https://github.com/auto-wcag/auto-wcag/pull/96